packages/flutter_bloc/test/bloc_listener_test.dart

```diff
@@ -72,6 +72,23 @@ class _MyAppState extends State<MyApp> {
 
 void main() {
   group('BlocListener', () {
+    testWidgets(
+        'throws AssertionError '
+        'when child is not specified', (tester) async {
+      const expected =
+          '''BlocListener<CounterCubit, int> used outside of MultiBlocListener must specify a child''';
+      await tester.pumpWidget(
+        BlocListener<CounterCubit, int>(
+          bloc: CounterCubit(),
+          listener: (context, state) {},
+        ),
+      );
+      expect(
+        tester.takeException(),
+        isA<AssertionError>().having((e) => e.message, 'message', expected),
+      );
+    });
```

Comment:
```suggestion
    },
    );
```

and format

Same goes for other tests, though I'm not sure what you're styling convention is here so feel free to ignore! 😄

Reply:
For internal consistency I prefer to leave the test format as is but thanks for the review, really appreciate it!